SYCL/ESIMD/api/functional/ctors/ctor_default.cpp

```diff
@@ -84,52 +82,53 @@ struct const_ref {
   }
 };
 
-// Functor with the main test routine to iterate over the pre-determined
-// datatypes
-template <typename DataT, int NumElems, typename TestCaseT> struct test {
-  bool operator()(sycl::queue &queue, const std::string &data_type) {
-    bool passed{true};
-
+// Struct that calls simd in provided context and then verifies obtained result.
+struct Verifier {
+  template <typename DataT, int NumElems, typename TestCaseT>
+  static bool verify(sycl::queue &queue, const std::vector<DataT> ref_data,
```

Comment:
```suggestion
  static bool verify(sycl::queue &queue, const std::vector<DataT>,
```
We are not using it, right? One more reason to have the `Verifier` and `run_test` as the single instance.

Moreover, how many times will this function run with, for example, `simd<int,1>` and `initializer`? Looks like it would loop over several times in the `run_test`, wouldn't it?

Reply:
The `Verifier` in test on simd default constructor was replaced into `run_test` in this commit: https://github.com/intel/llvm-test-suite/pull/614/commits/9fa6675d9d2aa91003819b23c03915291ea76af2